packages/docs/components/field-labels.md

```diff
@@ -116,7 +116,7 @@ Odyssey assumes inputs are required by default. Optional labels should be used t
         <option value="sirius-a">Sirius A</option>
         <option value="sirius-b">Sirius B</option>
       </select>
-      <label class="ods-label" for="overview-optional">Destination star <span class="ods-label--optional">Optional</span></label>
+      <label class="ods-field--label" for="overview-optional">Destination star <span class="ods-field--label--optional">Optional</span></label>
```

Comment:
I noticed that we place `Optional` text a child to the label itself, meaning that if the text is clicked it drives focus into the input. This isn't true of hint or error information based on the current design. This feels incorrect and I would think that clicking on any input related text should drive focus to the input.

Reply:
So far, this is the desired behavior and a broad standard. @joejezowski-okta is investigating updating our form behaviors, though, and I'll update this based on his findings.